payara/Payara
# FISH-6500 hazelcast-configuration-file Domain Property Ignored

## Description
Payara seems to completely ignore hazelcast config file

## Testing
### New tests
None

### Testing Performed
1. Build the project: mvn clean install -DskipTests
2. Start Payara Server
3. Go to localhost:4848 and click on Data Grid
4. Save the file Hazelcast-test.xml available at: https://payara.atlassian.net/browse/FISH-6500
5. Fill input: "Override configuration file" with the path of "Hazelcast-test.xml"
6. Click on Save button
7. Check content of "DAS Start Port"... It should be changed to 8876

### Testing Environment
Open JDK 1.8.0_312 on Windows 10 with Maven 3.8.4
